### PR TITLE
chore: minor changes to quickstart notebook

### PR DESCRIPTION
**Pull request by callumtilbury**
Apologies, I didn't get a chance to review the quickstart notebook in the previous PR. I finally had a proper read through, and there were a few typos.

I also found this line hard to parse:

> The configuration encompasses established hyperparameters, yet due to the utilisation of JAX in the implementation, certain hyperparameters may emerge with novel significance. Thus, here are the definitions for select questionable configurations:

https://github.com/instadeepai/Mava/blob/9a4c0aea8f2ae70f7d5ebf11033657049bb00d8d/examples/Quickstart.ipynb#L831

What does it mean? Could we perhaps reword it? (I think you wrote it originally, @OmaymaMahjoub?)

Thanks! :)

## What?
Minor text updates.

## Why?
Readability.

### CODE REVIEW

**Review by OmaymaMahjoub — DISMISSED** [review]
Thanks @callumtilbury for these changes!

**Comment by OmaymaMahjoub** [comment]
> I also found this line hard to parse:
> 
> > The configuration encompasses established hyperparameters, yet due to the utilisation of JAX in the implementation, certain hyperparameters may emerge with novel significance. Thus, here are the definitions for select questionable configurations:
> 
> https://github.com/instadeepai/Mava/blob/9a4c0aea8f2ae70f7d5ebf11033657049bb00d8d/examples/Quickstart.ipynb#L831
> 
> What does it mean? Could we perhaps reword it? (I think you wrote it originally, @OmaymaMahjoub?)

> —

Oh sorry I didn't read the conversation docs, tbh I forget why I added that line :sweat_smile: feel free to remove it and just mentioning there are config params jax_arch related and here is the definition to some of these questionable params

**Comment by callumtilbury** [comment]
Thanks @OmaymaMahjoub! Suggested a change—let me know if it's all right :)

**Review by OmaymaMahjoub — APPROVED** [review]
Thanks @callumtilbury LGTM!

**Review by WiemKhlifi — APPROVED** [review]
Thanks @callumtilbury 🔥   If @OmaymaMahjoub is happy then I'm too 🙌
